docker

```diff
@@ -0,0 +1 @@
+{"a": 1} exec -i vibrant_satoshi jq
```

Comment:
What is this?

Reply:
A missed artifact from debugging the integration test.  Revmoved

Reply:
Done in 3360a19